benchmarks/benchmarks/torch_benchmark/scattering1d.py

```diff
@@ -83,16 +83,13 @@ def setup(self, sc_params,  backend, device):
         # can take some time
 
     def time_constructor(self, sc_params,  backend, device):
-        if device == 'cuda':
-            torch.cuda.synchronize()
```

Comment:
Why don't we want to `synchronize`? Won't the timings be underestimated now?

Reply:
Non. This is the constructor for the Scattering. It doesn't involve cuda at all.